cluster/addons/dns/kube2sky/kube2sky.go

```diff
@@ -195,7 +195,7 @@ func newKubeClient() (*kclient.Client, error) {
 		var err error
 		if config, err = kclientcmd.NewNonInteractiveDeferredLoadingClientConfig(
 			&kclientcmd.ClientConfigLoadingRules{ExplicitPath: *argKubecfgFile},
-			&kclientcmd.ConfigOverrides{ClusterInfo: kclientcmdapi.Cluster{Server: masterUrl, InsecureSkipTLSVerify: true}}).ClientConfig(); err != nil {
+			&kclientcmd.ConfigOverrides{ClusterInfo: kclientcmdapi.Cluster{Server: masterUrl}}).ClientConfig(); err != nil {
```

Comment:
I'm not a huge fan of the way that the masterUrl is plumbed through now that we are using a kubeconfig file. Would it be possible to create an override struct that is initially empty and only gets the Server field overridden if --kube_master_url is explicitly set? Then we could just not bother setting it and control the apiserver location directly through the kubeconfig file.